src/components/molecules/CreateVenueModal/CreateVenueModal.scss

```diff
@@ -0,0 +1,80 @@
+@import "scss/constants.scss";
+
+.modal-backdrop.in {
+  background-color: $black;
+}
+
+.modal-content.in {
+  background-color: $saturated-black;
+}
```

Comment:
Sadly, it wasn't described how this works, only that "it works". Your guess is as good as mine.

Reply:
Checkout the link I shared in #tech recently about CSS specificity (called something like specifishitty). It’s a useful reference sheet that explains how it works/how to easily increase the specificity of things

Reply:
Can we add a comment to these explaining why they are as they are (eg. To increase the specificity to override the other styles). Otherwise I expect we’ll accidentally refactor the code in future and break this, as it deviates from our standard patterns